lib/robot.js

```diff
@@ -25,6 +25,25 @@ class Robot {
     });
   }
 
+  addEventToListeners(event, callback) {
+    const [name, action] = event.split('.');
+
+    this.events.on(name, async event => {
+      if (!action || action === event.payload.action) {
+        try {
+          const github = await this.auth(event.payload.installation.id);
+          const context = new Context(event, github);
+          await callback(context, context /* DEPRECATED: for backward compat */);
```

Comment:
@bkeepers `(context, event)` was deprecated in version `0.7.0`, we are currently on version `0.9.1`, do you think we should remove this for `0.10.0`?